examples/core/quad/main.rs

```diff
@@ -173,15 +211,15 @@ fn main() {
     });
 
     pipeline_desc.attributes.push(pso::AttributeDesc {
-        location: 1,
+        location: 0,
```

Comment:
doesn't this make the existing Vk shaders incompatible?

Reply:
No, it currently was wrongly implemented, switching uv and pos. Vulkan hasn't used the location but only the order they have been added to the attributes list. I think I included a fix for vulkan in this PR.